.github/ISSUE_TEMPLATE/new-ontology-request.md

```diff
@@ -52,17 +52,17 @@ GitHub username:
 
 ## Metadata
 
-Please fill in the following metadata record which will be used by the OBO Foundry website.
+Please fill in the following metadata record which will be used by the OBO Foundry website. Note that all the values are all examples in nature, for example `yourfourletterid` could be something like `aism`, `cohm`, `hpo` (it does not have to be four letters). `your_domain_like_for_example_anatomy` could be simply `anatomy`, and the license should be whatever your actual license is. An example can be found [here](https://github.com/OBOFoundry/OBOFoundry.github.io/edit/master/ontology/amphx.md), but you really only need to fill in the metadata mentioned here.
```

Comment:
Note that all the values are all examples in nature ->
Note that the values shown are just examples

> (it does not have to be four letters)
then why not make it something like "yourshortID"? i think we want to discourage the use of 2 or 3-letter IDs, though, right? so probably best not to show HPO as an example? (it is grandfathered)

Reply:
Thanks! Fixed these I think!